src/includes/localeGetters.ts

```diff
@@ -54,3 +54,42 @@ export const getLocaleFromHash = (hash: string) => {
 
   return getFromQueryString(window.location.hash.substr(1), hash);
 };
+
+export const getLocaleFromAcceptLanguageHeader = (header: string | null, availableLocales?: string[]): string | undefined => {
```

Comment:
Could you add a comment with some examples of what locale would be returned for a few headers. E.g
```js
getLocaleFromAcceptLanguageHeader('en-GB,en;q=0.9,es-ES;q=0.8,es;q=0.7,en-US;q=0.6') // => 'en-GB'
getLocaleFromAcceptLanguageHeader('en-GB,en;q=0.9,es-ES;q=0.8,es;q=0.7,en-US;q=0.6', ['es-ES', 'en-us']) // => 'es-ES'
getLocaleFromAcceptLanguageHeader('en-GB,en;q=0.9,es-ES;q=0.8,es;q=0.7,en-US;q=0.6', ['es', 'en-us']) // => 'en-US' (I'm actually not sure if in this case it should prefer `'es'` as it's the base language with the highest `q` or `en-US` because it's a full match
```

Reply:
After reading this, the best way to document this kind of stuff is with a test that actually calls the function and asserts the return values are what we expect.

Reply:
I've added a test